### PR TITLE
Add missing replicator elements ca_cert, cert, and key

### LINKED ISSUE #1937: Etcd replicator reference is missing fields
## This is an issue with:

- [ ] Bug (site functionality or styling)
- [ ] Errata (fix needed for doc content)
- [ ] New content (guide wanted)
- [x] Update (Add missing or refresh existing content)
- [ ] Enhancement (add new site functionality)

## Context
[Etcd replicator reference](https://docs.sensu.io/sensu-go/5.15/reference/etcdreplicators/) should contain all fields of an etcd replicator. See [EtcdReplicator struct](https://github.com/sensu/sensu-enterprise-go/blob/6322f6e4b553975d0b7009729410d56793b4d099/api/federation/v1/replicator.go#L21-L52) for the full spec. There should also be an example of a secure configuration, where insecure is false, and the key, cert, and ca cert fields are populated. This should be done for 5.15.0.

### PR DESCRIPTION
**Pull request by hillaryfraley**
## Description
Add `ca_cert`, `cert`, and `key` to etcd replicators reference

## Motivation and Context
Closes https://github.com/sensu/sensu-docs/issues/1937

### CODE REVIEW

File: content/sensu-go/5.15/reference/etcdreplicators.md [hunk 1]
```diff
@@ -61,12 +61,15 @@ example      | {{< highlight shell >}}api_version: federation/v1{{< /highlight >
 
 metadata     |      |
 -------------|------
-description  | Top-level scope that contains the replicator `name`.
+description  | Top-level scope that contains the replicator `name`, `ca_cert`, `cert`, and `key`.
```
Comment (nikkictl) [inline content/sensu-go/5.15/reference/etcdreplicators.md]: These fields are not apart of the metadata, rather they are in the spec.

Comment (hillaryfraley) [reply content/sensu-go/5.15/reference/etcdreplicators.md]: I updated them.

File: content/sensu-go/5.15/reference/etcdreplicators.md [hunk 2]
```diff
@@ -94,6 +97,27 @@ example      | {{< highlight shell >}}name: my_replicator{{< /highlight >}}
...
+required     | true
+type         | String
+example      | {{< highlight shell >}}cert: /path/to/ssl/cert.pem{{< /highlight >}}
+
+key          |      |
+-------------|------
+description  | Path to the PEM-format key file associated with the `cert` to use for TLS client authentication.
+required     | true
+type         | String
+example      | {{< highlight shell >}}key: /path/to/ssl/key.pem{{< /highlight >}}
```
Comment (nikkictl) [inline content/sensu-go/5.15/reference/etcdreplicators.md]: These fields are only required if `insecure = false` which is the default configuration. However, if `insecure = true` these fields are not required.

File: content/sensu-go/5.15/reference/etcdreplicators.md [hunk 3]
```diff
@@ -94,6 +97,27 @@ example      | {{< highlight shell >}}name: my_replicator{{< /highlight >}}
 
 #### Spec attributes
 
+ca_cert      |      |
+-------------|------
+description  | Path to an the PEM-format CA certificate to use for TLS client authentication.
+required     | true
```
Comment (hillaryfraley) [inline content/sensu-go/5.15/reference/etcdreplicators.md]:
```suggestion
required     | true if `insecure: false` (which is the default configuration). If `insecure: true`, `ca_cert` is not required.
```

File: content/sensu-go/5.15/reference/etcdreplicators.md [hunk 4]
```diff
@@ -94,6 +97,27 @@ example      | {{< highlight shell >}}name: my_replicator{{< /highlight >}}
...
+type         | String
+example      | {{< highlight shell >}}ca_cert: /path/to/trusted-certificate-authorities.pem{{< /highlight >}}
+
+cert         |      |
+-------------|------
+description  | Path to the PEM-format certificate to use for TLS client authentication.
+required     | true
```
Comment (hillaryfraley) [inline content/sensu-go/5.15/reference/etcdreplicators.md]:
```suggestion
required     | true if `insecure: false` (which is the default configuration). If `insecure: true`, `cert` is not required.
```

File: content/sensu-go/5.15/reference/etcdreplicators.md [hunk 5]
```diff
@@ -94,6 +97,27 @@ example      | {{< highlight shell >}}name: my_replicator{{< /highlight >}}
...
+type         | String
+example      | {{< highlight shell >}}cert: /path/to/ssl/cert.pem{{< /highlight >}}
+
+key          |      |
+-------------|------
+description  | Path to the PEM-format key file associated with the `cert` to use for TLS client authentication.
+required     | true
```
Comment (hillaryfraley) [inline content/sensu-go/5.15/reference/etcdreplicators.md]:
```suggestion
required     | true if `insecure: false` (which is the default configuration). If `insecure: true`, `key` is not required.
```

**Review by nikkictl — APPROVED** [review]

**Pull request merged**